packages/codepush/src/__tests__/index.test.tsx

```diff
@@ -101,7 +102,7 @@ describe('AppCenter Codepush integration', () => {
                 'env',
                 'appId',
                 true,
-                true,
+                false,
```

Comment:
is this change intentional?

Reply:
If we enable resource tracking, we now have to provide a mock for XMLHttpRequest in the tests.
This is opens up for more reasons for these tests to break, and the codepush integration is not related at all to the resource tracking.
So I figured it was best to simply disable this option, as it does not impact the coverage of the tests and needs less code than maintaining an XMLHttpRequest mock